components/telegram/sources/channel-updates/channel-updates.js

```diff
@@ -0,0 +1,45 @@
+const telegram = require("../../telegram.app.js");
+
+module.exports = {
+  key: "telegram-channel-updates",
+  name: "Channel Updates (Instant)",
+  description: "Emits an event each time a channel message is created or updated.",
+  version: "0.0.1",
+  props: {
+    db: "$.service.db",
+    http: {
+      type: "$.interface.http",
+      customResponse: true,
+    },
+    telegram,
+  },
+  hooks: {
+    async activate() {
+      const response = await this.telegram.createHook(this.http.endpoint, ['channel_post', 'edited_channel_post']);
+      console.log(response.data.description);
+    },
+    async deactivate() {
+      const response = await this.telegram.deleteHook();
```

Comment:
need to get the hookId from db and pass it to `deleteHoook()`

Reply:
There is no hookId to pass.